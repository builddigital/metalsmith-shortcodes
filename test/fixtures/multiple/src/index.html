---
date: 2014-03-29
---

<p>Here's foo!</p>

[foo klass=bar]

<p>And here's bar!</p>

[bar]tomatoes,paperclips,raccoons,trampolines[/bar]

<p>And here's another foo!</p>

[foo klass=baz]


