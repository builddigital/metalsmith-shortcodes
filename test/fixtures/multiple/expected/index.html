
<p>Here's foo!</p>

<div class="bar"><foo data-date="2014-03-29"></foo></div>

<p>And here's bar!</p>

<ul><li>tomatoes</li><li>paperclips</li><li>raccoons</li><li>trampolines</li></ul>

<p>And here's another foo!</p>

<div class="baz"><foo data-date="2014-03-29"></foo></div>


